grandnode/grandnode2
# State not showing in pickup address

Resolves #issueNumber  
Type: **bugfix**

## Issue
pickup point address state is not displaying during checkout.

## Solution
using StateProvinceId fetch state name and display in UI.

## Breaking changes
none

## Testing
1. Add pickup point with address
2. start checkout 
3. choose pickup option and verify state is showing in pickup address
